README.md

```diff
@@ -58,21 +60,28 @@ To run the application, use the `main.py` file. Below are some example usages.
 ### Generate Yearly Extreme Report
 For a given year, display the highest temperature and day, lowest temperature and day, most humid day, and humidity.
 ```sh
+
 python3 main.py /path/to/weather-data-txt-files-dir -e 2004
```

Comment:
Path should be picked automatically, applies to all

Reply:
done